release/dist/cli/cli.go

```diff
@@ -78,6 +79,20 @@ func CLI(getTargets func(unixpkgs.Signers) ([]dist.Target, error)) *ffcli.Comman
 			Filters can use glob patterns (* and ?).
 			`),
 			},
+			{
+				Name: "gen-key",
+				Exec: func(ctx context.Context, args []string) error {
+					return runGenKey(ctx)
+				},
+				ShortUsage: "dist gen-key",
+				ShortHelp:  "Generate root or sining key pair",
+				FlagSet: (func() *flag.FlagSet {
+					fs := flag.NewFlagSet("gen-key", flag.ExitOnError)
+					fs.StringVar(&genKeyArgs.privPath, "priv-path", "key.priv", "output path for the private key")
```

Comment:
Since the files will be PEM, maybe the default values could use the .pem file extension? private-key.pem, public-key.pem, or the like. Up to you.